src/style.css

```
@@ -1 +1,68 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  position: sticky;
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  margin-left: 50px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  cursor: pointer;
+  text-decoration: none;
+  display: flex;
+  position: relative;
+  text-transform: uppercase;
+  padding: 23px 0;
+  color: #000;
+  line-height: 14px;
+  font-size: 12px;
+  font-weight: 500;
+  transition: all 0.5s;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
```

don't increase specificity if it's unnecessary
```suggestion
.is-active {
```